### DISCUSSION TITLE
How can I save these values of heights as a custom style?

### DISCUSSION THREAD

**Discussion by Korb** [openstyles/stylus#1410] [Q&A] [answered]
![156368136-585213a2-7e45-4cc1-8895-c72530ef6ae9](https://user-images.githubusercontent.com/3509797/156371908-ac2ab964-a99d-4ab7-8a3f-4f4289084832.png)

"Copy" / "The outer HTML":
`<div style="height: 716px;" class="bx-messenger-body"><div class="bx-messenger-body-bg"><div class="bx-messenger-body-wrap"><span class="bx-notifier-content-link-history bx-notifier-content-link-history-empty"><span class="bx-notifier-item-button bx-notifier-item-button-white"> (...)`
and
`<textarea style="height: 143px;" class="bx-messenger-textarea-input"></textarea>`
respectively.

In the inspector, I set the desired values for the size of these fields. They look great in the browser. But, of course, are reset after page refresh. Unfortunately, I don't understand anything about cascading styles. And never created custom styles to figure out exactly what a custom style should look like to connect it to Stylus?

**Comment by decembre** [comment, marked as answer]
To quickly help you:
Copy that in a new userstyle:

```
@namespace url(http://www.w3.org/1999/xhtml);

@-moz-document domain("EXAMPLE.com") {

/* TEST */
.bx-messenger-body {
    height: 745px !important;
}
.bx-messenger-textarea textarea {
    height: 114px !important;
}

/* === END  === */
}
```

In fact, you can make change in the inspector (in the right panel) And copy these changes in your userstyle to save them.
if i remember well (I don't use it)t you can use too, in the devtool, its style editor:
if i remember it can save your change to make an userstyle.

Someone one here (or you) here can surely give to you some basic links to understand some basic thinks about CSS and Userstyles...
And they are many other sites which can help you.
I hope you can play more in future with CSS:
that's funny.
